setup.py

```diff
@@ -18,7 +18,7 @@
 EMAIL = "aljparr0@gmail.com"
 AUTHOR = "Adam Jacquier-Parr"
 REQUIRES_PYTHON = ">=3.6.0"
-VERSION = "0.1.0"
+VERSION = "0.2.0"
```

Comment:
We might want to bump this to `1.0.0` in this PR since we are probably introducing breaking changes.